src/database/seeders/20200109111519-demo-user.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert(
+      'Users',
+      [
+        {
+          id: '40b6c719-68b2-4097-95ea-6fe230784e29',
+          name: 'John Doe',
+          email: 'john.doe@gmail.com',
+          username: 'john-doe',
+          password: 'pass',
+          verified: false,
+        },
+      ],
+      {},
```

Comment:
Can we seed one more user here, who will be a `Super Admin` as the above user is already linked to an organization?